.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Description
+
+
+### Version
+**fundamental-react:**
+**fiori-fundamentals:**
+
+---
+
+_**NOTE:** Where applicable, please include uncropped screen captures._
+
+_**DISCLAIMER:**
+After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._
```

Comment:
Looks like there's an extra _team_ after if more _information is needed, the fundamental-react team_ now.